### PR TITLE
Add support for VB.NET

### PR DESCRIPTION
**Pull request by weloytty**
I have a lot of VB.NET code that could take advantage of this, so I added support for vb.net.  I duplicated all of the unit tests and they all pass, and the output is syntactically correct in vb.

The only shared code I modified was DumpStyle.cs, to add VisualBasic to the DumpStyle enum and TypeExtensions.cs, where I added GetFormattedNameVB to handle vb's generics.

### CODE REVIEW

**Comment by thomasgalliker** [comment]
Wow ok. Thanks for your contribution. I will have a closer look at it soon (~next week). I can‘t verify the VB syntax but I‘ll review it anyway. Before I release another nuget -pre version, I‘ll be releasing the next stable version to nuget. Hope you re ok with this.

File: ObjectDumper.sln [hunk 1]
```diff
@@ -191,13 +193,62 @@ Global
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x64.Build.0 = Release|Any CPU
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x86.ActiveCfg = Release|Any CPU
 		{CF00093C-CB18-4400-8A83-8090DE38D7C7}.Release|x86.Build.0 = Release|Any CPU
+		{C6FDB4C9-8E2D-44E3-8C59-0968C98D8BFE}.Ad-Hoc|Any CPU.ActiveCfg = Release|Any CPU
```
Comment (thomasgalliker) [inline ObjectDumper.sln]: @weloytty are these intented changes? Not sure if all these build configurations are needed..

Comment (weloytty) [reply ObjectDumper.sln]: No -- I believe they are a result of me using VS2019 to do development, and none are needed.

**Comment by thomasgalliker** [comment]
The reason why I have to reject this PR: I'm personally not able to support VB .NET and I see lack of community support for such things. Thanks anyway for the idea & the effort.

**Pull request closed without merging**